### DISCUSSION TITLE
1.20.11381 - Where did it come from?

### DISCUSSION THREAD

**Discussion by OranguTech** [microsoft/terminal#17375] [General]
Updated via the Store, to ver  `1.20.11381`, release doesn't exist here (or the tag). I checked the [winget-pkgs repo](https://github.com/microsoft/winget-pkgs/commits/master/manifests/m/Microsoft/WindowsTerminal) too for good measure.

**Comment by zadjii-msft** [comment]
@DHowett HA someone noticed. 

We pushed a hotfix a couple weeks ago, but apparently forgot to push the tag. We were just yesterday bemusing that no one seemed to notice!

**Reply by OranguTech** [reply]
(I'm sure I'm not the only one, I was just the first to be pedantic/obsessive/bored enough to poke at it. :)
